.goreleaser.yml

```diff
@@ -34,3 +34,5 @@ brew:
     description: Command-line interface for Banzai Cloud Pipeline platform
     test: |
         system "#{bin}/banzai --version"
+    dependencies:
+        - kubectl
```

Comment:
this is an alias, the canonical name of the package is `kubernetes-cli`. does this work this way?

Reply:
TBH I have never installed it with kubernetes-cli, can we try this out?

Reply:
seems ok 

```diff --git a/Formula/banzai-cli.rb b/Formula/banzai-cli.rb
index f3d76ce..668f31e 100644
--- a/Formula/banzai-cli.rb
+++ b/Formula/banzai-cli.rb
@@ -4,6 +4,7 @@ class BanzaiCli < Formula
   url "https://github.com/banzaicloud/banzai-cli/releases/download/0.0.19/banzai_0.0.19_darwin_amd64.tar.gz"
   version "0.0.19"
   sha256 "11e4f1a967f0cb89309d5c16271cce104a5183794308726164e9d19ed8c50bbf"
+  depends_on "kubectl"

   def install
     bin.install "banzai"
```

```
Mate% brew install banzaicloud/tap/banzai-cli                                                                                                                        [/usr/local/Homebrew/Library/Taps/banzaicloud/homebrew-tap] [master*]
==> Installing banzai-cli from banzaicloud/tap
==> Installing dependencies for banzaicloud/tap/banzai-cli: kubectl
==> Installing banzaicloud/tap/banzai-cli dependency: kubectl
==> Downloading https://homebrew.bintray.com/bottles/kubernetes-cli-1.15.0.mojave.bottle.tar.gz
Already downloaded: /Users/maat/Library/Caches/Homebrew/downloads/8536301089a3d302a0913dfd27c0a16e10f27c0eda3d4652205d60c63da0a1da--kubernetes-cli-1.15.0.mojave.bottle.tar.gz
==> Pouring kubernetes-cli-1.15.0.mojave.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/kubernetes-cli/1.15.0: 234 files, 47.9MB
==> Installing banzaicloud/tap/banzai-cli```